src/checkPassword.test.js

```diff
@@ -8,11 +8,39 @@ describe(`Function 'checkPassword':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof checkPassword('')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('Password1!')).toBe(true);
+  });
+
+  it(`should return 'false' if password has more than 16 characters`, () => {
+    expect(checkPassword('Password1!qwertyu')).toBe(false);
+  });
+
+  it(`should return 'false' if password has less than 8 characters`, () => {
+    expect(checkPassword('qwerty')).toBe(false);
```

Comment:
```suggestion
    expect(checkPassword('Qwer1!')).toBe(false);
```
дотримуємося всіх правил, окрім одного